_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,251 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
```

Comment:
what is this for?

Reply:
I don't know, I actually have no idea what this metadata is for or if it's needed. Maybe you can find if it talks about it here: https://help.github.com/categories/github-pages-basics/

Reply:
Hm, it seems to have something to do with how we set up our navigation bar - we must be iterating over the pages at some point in order to list them in the nav bar. When I remove it, I see errors asking me to set the order. [This stack overflow post](https://stackoverflow.com/questions/13266369/how-to-change-the-default-order-pages-in-jekyll) seems relevant. 

tl;dr: leaving it in for now, the "15" is arbitrary and it seems we just need some number there, but something to keep in mind if and when we refactor the Grafeas site at some point.